### PR TITLE
[5.0.x] Apply multiple module project to 5.0.x #494

### PR DESCRIPTION
**Pull request by btyanagawamg**
Please review #494 .

### CODE REVIEW

File: terasoluna-tourreservation-selenium/pom.xml [hunk 1]
```diff
@@ -2,20 +2,27 @@
...
+    </parent>
 
     <dependencies>
         <dependency>
             <groupId>${project.groupId}</groupId>
+            <artifactId>terasoluna-tourreservation-env</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-selenium/pom.xml]: - [ ] Why you changed dependency? I think that it is sufficient to change the project structure.
If you change this, please see and consider to change 5.1 and later too.

Comment (btyanagawamg) [inline terasoluna-tourreservation-selenium/pom.xml]: I removed `env` dependency at 1a915e7838d607f7aefe5f5257c3d237277721a4 cpmmit .

**Comment by yoshikawaa** [comment]
@btyanagawamg 
Please rebase and resolve conflicts!

**Comment by btyanagawamg** [comment]
@yoshikawaa 
I rebased and resolved conflicts .

File: terasoluna-tourreservation-web/pom.xml [hunk 2]
```diff
@@ -46,13 +47,12 @@
         <!-- == Begin TourReservation == -->
         <dependency>
             <groupId>${project.groupId}</groupId>
+            <artifactId>terasoluna-tourreservation-env</artifactId>
+            <scope>provided</scope>
+        </dependency>
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/pom.xml]: - [ ] It is wrong that domain depends on env. 
Please see [master/domain](https://github.com/terasolunaorg/terasoluna-tourreservation/blob/master/terasoluna-tourreservation-domain/pom.xml#L18-L19) and [master/web](https://github.com/terasolunaorg/terasoluna-tourreservation/blob/master/terasoluna-tourreservation-web/pom.xml#L49-L53), or [blank-5.0/domain](https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/blob/5.0.x/projectName-domain/pom.xml#L23-L24) and [blank-5.0/web](https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/blob/5.0.x/projectName-web/pom.xml#L72-L76).

**Comment by yoshikawaa** [comment]
@btyanagawamg 
Fix errors on Travis CI !

File: terasoluna-tourreservation-web/pom.xml (hunk 2 above)
Comment (btyanagawamg) [inline terasoluna-tourreservation-web/pom.xml]: I removed env dependency at web project at cd1fc360d095689f8fc086739b54caee2e8d12a9 commit .
But domain depends on env at [master/domain](cd1fc360d095689f8fc086739b54caee2e8d12a9) .

Comment (yoshikawaa) [inline terasoluna-tourreservation-web/pom.xml]: > But original 5.0.x/domain project depends on env project and 5.0.x project cant work if env scope change for test in domain pom .

Please investigate the cause.
You should not make lazy corrections, please respect the thought of the blank project.

Comment (yoshikawaa) [inline terasoluna-tourreservation-web/pom.xml]: @btyanagawamg 
See [Changes](https://github.com/terasolunaorg/terasoluna-tourreservation/compare/issues/494_apply_multiple_module_project_5.0.x...issues/494_apply_multiple_module_project_5.0.x-confirm?expand=1), and [Result of Travis CI](https://travis-ci.org/terasolunaorg/terasoluna-tourreservation/builds/267724944?utm_source=github_status&utm_medium=notification).
If only change scope in domain, Logback.xml in env does not apply and a lot of logs are output.

Comment (btyanagawamg) [reply terasoluna-tourreservation-web/pom.xml]: I confirmed the comments above and cherry-picked your commit ( 48b295675cd912250e215693f6000e6aa9afe91d )(and I changed the commit message).

**Comment by yoshikawaa** [comment]
@btyanagawamg 
Please backport and sideport.

**Pull request merged**